packages/uswds/src/Widgets/URLWidget.tsx

```diff
@@ -0,0 +1,10 @@
+import { FormContextType, RJSFSchema, StrictRJSFSchema, WidgetProps } from '@rjsf/utils';
+import TextInputWidget from './TextInputWidget'; // Corrected import
+
+export default function URLWidget<
```

Comment:
The `core` implementation simply uses the `BaseInputTemplate` with the `type='url'` So you can just delete this if you've implemented the `BaseInputTemplate` properly